### ISSUE TITLE
When adding new columns to a table add an option to sort table columns in the order of the underlying data source

### ISSUE THREAD

**Issue by jasonhorner** [otykier/TabularEditor#935]
**Is your feature request related to a problem? Please describe.**
When adding columns by default they get added to the end of the list of table columns.  

**Describe the solution you'd like**
I'd like the option to reorder the columns in the model as they are in the underlying data source and optionally allow for alphabetical sorting as well.

**Describe alternatives you've considered**
Manually editing the underlying JSON

**Additional context**
This is similar to how the SSAS MD replace table worked

**Comment by jasonhorner** [comment]
This is correct. however I find that it easier to work with the model columns in the order defined in my tables / views: Pk, SK's, Natural Keys and attributes and then measures.

**Comment by otykier** [comment]
While I don't have any plans to allow reordering through the UI, you can certainly edit the JSON manually (through a text editor). You can already toggle between alphabetical/metadata ordering within Tabular Editor by pressing CTLR+F2.
